### PR TITLE
[test, otbn] Switch milestone to stage for newly added test

### PR DESCRIPTION
**Pull request by vogelpi**

### CODE REVIEW

**Comment by vogelpi** [comment]
We recently aligned some terms, e.g. `milestone` is now `stage`. PR #14319 got last updated before this change but merged afterwards. That's why CI is currently broken.

**Review by ctopal — APPROVED** [review]

**Comment by vogelpi** [comment]
Thanks for your approval @ctopal . Private CI has now passed. I am merging this to unblock CI.

**Pull request merged**
